assets/js/renderer.js

```
@@ -1,186 +1,179 @@
-      result = "";
-    }
-  
-    if (result) {
-      document.getElementById(label_id).classList.add(tip_color);
-      document.getElementById(label_id).innerText = result;
-      setTimeout(function () {
-        remove_label_text(label_id);
-      }, 5000);
-    }
-  };
+  let count = parseInt(document.getElementById('key-count').value)
```

ESLint

[error] 'count' is never reassigned. Use 'const' instead.

(prefer-const)